config/isr.py

```diff
@@ -1,7 +1,6 @@
 # Config override for lsst.ip.isr.IsrTask
 
 # Can't ingest the necessary datasets
-config.doBrighterFatter = False
 config.doAttachTransmissionCurve = False
 config.doUseOpticsTransmission = False
 config.doUseFilterTransmission = False
```

Comment:
Given the comment above (and the fact that presumably removing the line doesn't break things), I wonder how many of the other lines are also unneccessary. I think this was all written before the Gen 2 curated calibs system existed.

Reply:
Actually, I'm confused by this change. This line should have applied to *both* Gen 2 and Gen 3, so why was Gen 3 being run with brighter-fatter?